### PR TITLE
Add grouped convolution changes to changelog

### PR DESCRIPTION
**Pull request by bartekxk**

### CODE REVIEW

**Review by illsilin — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,16 +5,23 @@ Full documentation for Composable Kernel is not yet available.
...
 
 ### Fixed
+ - Fixed a hazard with using inline v_dot (#808).
+ - Fixed a bugs in grouped Convolution backward data with not padded K (#848 #876).
 
 ### Optimizations
 
 ### Added
 - Added image to column (#867) and column to image kernels (#930).
+- Added support for 3D grouped Convolution forward on Navi3x (#935).
```
Comment (saadrahim) [inline CHANGELOG.md]: Navi3x is an internal term. Navi3 GPU family? @LisaDelaney we should standardize this. Please add to spell check if possible I think a tm may be needed as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,16 +5,23 @@ Full documentation for Composable Kernel is not yet available.
 ## (Unreleased) CK for ROCm 6.0.0
 
 ### Fixed
+ - Fixed a hazard with using inline v_dot (#808).
+ - Fixed a bugs in grouped Convolution backward data with not padded K (#848 #876).
```
Comment (saadrahim) [inline CHANGELOG.md]: Fixed _two_ bugs in

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,16 +5,23 @@ Full documentation for Composable Kernel is not yet available.
...
 
 ### Optimizations
 
 ### Added
 - Added image to column (#867) and column to image kernels (#930).
+- Added support for 3D grouped Convolution forward on Navi3x (#935).
+- Added support for small K and C for grouped Convolution (#822 #879 #897).
+- Added support NHWGC 2D and 3D grouped Convolution backward weight (#769 #804).
+- Added support bf16/f32/f16 and NHWGC 2D and 3d grouped Convolution backward data (#757 #799).
```
Comment (saadrahim) [inline CHANGELOG.md]: too many ands, missing comma likely. @LisaDelaney please verify

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,16 +5,23 @@ Full documentation for Composable Kernel is not yet available.
...
+- Added support for Batched Gemm DL (#732).
 
 ### Changed
+ - Changed grouped Convolution API to keep consistency with other convolution kernels (#817).
```
Comment (saadrahim) [inline CHANGELOG.md]: capitalization of convolution is not consistent. @LisaDelaney please suggest standard.

**Review by saadrahim — CHANGES_REQUESTED** [review]
@LisaDelaney hoping you can take the lead on the review and make sure my suggestions are consistent on your edits on other projects.

File: CHANGELOG.md (hunk 1 above)
Comment (LisaDelaney) [reply CHANGELOG.md]: > Navi3x is an internal term. Navi3 GPU family? @LisaDelaney we should standardize this. Please add to spell check if possible I think a tm may be needed as well.

Looks like maybe AMD RDNA 3?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,43 +1,52 @@
...
 Full documentation for Composable Kernel is not yet available.
 
 ## (Unreleased) CK for ROCm 6.0.0
 
-### Fixed
+### Fixes
+ - Fixed a hazard associated with inline v_dot (#808).
+ - Fixed a bug in grouped convolution backward data without K padding (#848 #876).
 
 ### Optimizations
```
Comment (LisaDelaney) [inline CHANGELOG.md:13]: Should we add 'None' below empty headers?

File: CHANGELOG.md (hunk 2 above)
Comment (bartekxk) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (bartekxk) [reply CHANGELOG.md]: Done by @LisaDelaney

File: CHANGELOG.md (hunk 4 above)
Comment (bartekxk) [reply CHANGELOG.md]: Done by @LisaDelaney

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,43 +1,54 @@
...
 
+## CK 0.2.0 for ROCm 5.7.0
 
-## CK 0.2.0 for ROCm 5.5.0
-
-### Fixed
+### Fixes
 - Fixed a bug in 6-dimensional kernels (#555).
-- Fixed grouped ConvBwdWeight test case failure (#524).
+- Fixed a test case failure with grouped ConvBwdWeight (#524).
```
Comment (bartekxk) [inline CHANGELOG.md]:
```suggestion
- Fixed a test case failure with grouped convolution backward weight (#524).
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,43 +1,54 @@
...
+- Added support for AMD RDNA 3.
+- Added a user tutorial (#563).
+- Added additional instances for irregular GEMM sizes (#560).
+- Added an inter-wave consumer-producer programming model for GEMM kernels (#310).
 - Added multi-D GEMM client APIs (#534).
 - Added multi-embeddings support (#542).
-- Added Navi3x blockwise GEMM and real GEMM support (#541).
-- Added Navi grouped ConvBwdWeight support (#505).
+- Added AMD RDNA 3 blockwise GEMM and real GEMM support (#541).
+- Added AMD RDNA-grouped ConvBwdWeight support (#505).
```
Comment (bartekxk) [inline CHANGELOG.md]:
```suggestion
- Added AMD RDNA grouped convolution backward weight support (#505).
```

**Review by saadrahim — APPROVED** [review]

**Review by LisaDelaney — APPROVED** [review]

**Pull request merged**